.tractusx

```diff
@@ -0,0 +1,26 @@
+###############################################################
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################
+
+product: "SSI Dim Wallet Stub"
```

Comment:
DIM should be uppercase

Reply:
fixed with: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/1/commits/88c2441f820cfb0be2bb85246e33ae7235090154